file_paths.md

```diff
@@ -289,6 +290,9 @@ running as root and non-root, e.g. `puppet.conf` is always
     ~/.puppetlabs/var/log                 # :logdir                     ~/.puppet/var/log
     ~/.puppetlabs/opt/facter/facts.d      # n/a                         ~/.facter/facts.d
 
+On Windows, when not running on the SYSTEM account
```

Comment:
FYI, I double checked that `NetworkService` and `LocalService` use user-specific Temp directories, so `SYSTEM` does appear to be the only default account that writes to `C:\Windows\Temp`